File: src/com/l2kt/gameserver/network/clientpackets/RequestExFishRanking.java
package com.l2kt.gameserver.network.clientpackets;

import com.l2kt.Config;
import com.l2kt.gameserver.data.manager.FishingChampionshipManager;
import com.l2kt.gameserver.model.actor.instance.Player;

public final class RequestExFishRanking extends L2GameClientPacket
{
	@Override
	protected void readImpl()
	{
	}
	
	@Override
	protected void runImpl()
	{
		final Player player = getClient().getActiveChar();
		if (player == null)
			return;
		
		if (Config.ALT_FISH_CHAMPIONSHIP_ENABLED)
			FishingChampionshipManager.getInstance().showMidResult(player);
	}
}

File: src/com/l2kt/gameserver/network/clientpackets/RequestExFishRanking.kt
package com.l2kt.gameserver.network.clientpackets

import com.l2kt.Config
import com.l2kt.gameserver.data.manager.FishingChampionshipManager

class RequestExFishRanking : L2GameClientPacket() {
    override fun readImpl() {}

    override fun runImpl() {
        val player = client.activeChar ?: return

        if (Config.ALT_FISH_CHAMPIONSHIP_ENABLED)
            FishingChampionshipManager.getInstance().showMidResult(player)
    }
}